addons/evm/src/contracts/src/AtomicProxyDeploymentFactory.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
```

Comment:
Hmm. I don't know how I feel about having gitsubmodules. What would be the alternatives?

Reply:
Yeah, I don't like it either. This is the standard way for managing contract dependencies with foundry.

We could have a separate repo for our evm contracts, but that introduces a whole CI process for cloning another repo when we make a build.

Is there any problem with just copying out the contract?

Reply:
Let's copy the contract for now and backlog this as something we need to revisit?

Reply:
Done! Task in planner [here](https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.planner/mytasks?tenantId=0f1e1f86-ff36-4233-ba7c-e2d48a07bb41&webUrl=https%3A%2F%2Ftasks.teams.microsoft.com%2Fteamsui%2FpersonalApp%2Falltasklists&context=%7B%22subEntityId%22%3A%22%2Fv1%2Fpremiumplan%2Fc7a7d962-0836-4ed0-8a67-774535750432%2Forg%2Fdeb736b0-d91b-ef11-9f85-000d3a537223%2Fview%2Fboard%2Ftask%2FFB1E8DBC-10DA-4146-8580-175630BFBF04%22%7D)